docs/3.0/deploy/infrastructure-concepts/workers.mdx

```diff
@@ -102,6 +102,41 @@ Workers have two statuses: `ONLINE` and `OFFLINE`. A worker is online if it send
 If a worker misses three heartbeats, it is considered offline. By default, a worker is considered offline a maximum of 90 seconds 
 after it stopped sending heartbeats, but you can configure the threshold with the `PREFECT_WORKER_HEARTBEAT_SECONDS` setting.
 
+### Worker logs
+<Warning>This feature is experimental as of Prefect version 3.X.X</Warning>
+
+<span class="badge cloud"></span>Workers can now send logs directly to Prefect Cloud. To enable this feature:
+
+1. Set the environment variable:
+
+```bash
+PREFECT_EXPERIMENTS_WORKER_LOGGING_TO_API_ENABLED=True
+```
+
+Once enabled:
+- All worker logs are automatically sent to the Prefect Cloud API
+- Logs are accessible through both the Prefect Cloud UI and API
+- Each flow run log includes a direct link to its associated worker's logs
```

Comment:
Is this true? 🤔

Reply:
That seems mostly true, but I figured we'd have one print statement at the top of logging for the flow run that linked to the worker, while this suggests we'll literally include a link for each logs. I realize the implementation links the logs, but we probably don't need to mention that here. I'd describe it closer to what the user will see (assuming you and I are on the same page about that!):
- Each flow run will include a link to its associated worker's logs